rules-reviewed/eap7/eap71/hibernate51-53.windup.xml

```diff
@@ -60,25 +60,40 @@
                 <matches pattern="(hibernate-java8|hibernate-entitymanager)" />
             </where>
         </rule>
-        <!--https://issues.jboss.org/browse/WINDUPRULE-378
-        <rule id="hibernate51-53-00200">
-            <when>
-            </when>
-            <perform>
-            </perform>
-        </rule>
-        -->
-        <!--https://issues.jboss.org/browse/WINDUPRULE-379
+        <!--https://issues.jboss.org/browse/WINDUPRULE-379-->
         <rule id="hibernate51-53-00300">
             <when>
+                <javaclass references="org.hibernate.{className}">
```

Comment:
I think it's better to add the `<location>` condition in this rule because, in the end, the issue can be solved by adding just one property so having multiple incidents for each time an exception is referenced will generate too much incidents compared to the "one property to fix the all" solution
```suggestion
                <javaclass references="org.hibernate.{className}">
                    <location>IMPORT</location>
```